appimage/Makefile

```diff
@@ -0,0 +1,15 @@
+# in dev mode we just use the latest image as the start point
+ifneq ($(strip $(DEV_MODE)),)
+DOCKERFILE_HEAD = <(echo "FROM ns1labs/pktvisor:latest")
```

Comment:
not 100% necessary but it might be cool/useful to allow setting whether this is "latest" or e.g. "develop", or another specific tag.

Reply:
seems doable let me add that